specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2020-05-01-preview/cpim.json

```diff
@@ -0,0 +1,521 @@
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string",
+              "readOnly": true
+            },
+            "description": {
+              "description": "Friendly name of the operation",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of listing operations for the resourceProvider",
+      "properties": {
+        "value": {
+          "description": "List of cpim service operations supported by the Microsoft.AzureActiveDirectory resource provider.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    },
+    "GuestUsagesResourceProperties": {
+      "description": "Guest Usages Resource Properties",
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "An identifier that represents the Guest Usages Resource Properties."
+        }
+      }
+    },
+    "GuestUsagesResource": {
+      "description": "Guest Usages Resource",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "An identifier that represents the Guest Usages resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the Guest Usages resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of the Guest Usages resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the Guest Usages resource.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Key-value pairs of additional resource provisioning properties."
+        },
+        "properties": {
+          "$ref": "#/definitions/GuestUsagesResourceProperties",
+          "x-ms-client-flatten": true,
+          "description": "The Guest Usages Resource Properties"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "GuestUsagesResourceList": {
+      "description": "The collection of guest usages resources",
+      "properties": {
+        "value": {
+          "description": "List of guest usages resources",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/GuestUsagesResource"
+          }
+        }
+      }
+    },
+    "CloudError": {
```

Comment:
Why not simply reference the common error type, as here: https://github.com/Azure/azure-rest-api-specs/blob/9df7c71f9b955ea66fc9ede43842f937ca53a646/specification/resources/resource-manager/Microsoft.Resources/stable/2019-08-01/resources.json#L4232

Reply:
used that